source/shared/cpp/ObjectModel/HostConfig.cpp

```diff
@@ -75,6 +84,24 @@ HostConfig HostConfig::Deserialize(const Json::Value& json)
     return result;
 }
 
+FontSizesConfig::FontSizesConfig()
+{
+    smallFontSize = UINT_MAX;
```

Comment:
>UINT_MAX [](start = 20, length = 8)

we should be using the stl helpers for this: [https://en.cppreference.com/w/cpp/types/numeric_limits](https://en.cppreference.com/w/cpp/types/numeric_limits)

Reply:
in helping Carlos with this change, it became apparent that it would be more trouble than it's worth (problems with syntax and swig).

---
In reply to: [227140436](https://github.com/Microsoft/AdaptiveCards/pull/2101#discussion_r227140436) [](ancestors = 227140436)